acceptance/tests/pxp_agent_version.rb

```diff
@@ -4,7 +4,8 @@
 
 agents.each do |agent|
   step 'cd into pxp-agent bin folder and check the version' do
-    on(agent, "cd #{pxp_agent_dir(agent)} && ./pxp-agent --version") do |result|
+    # Vendored Ruby needs to be on path. The "privatebindir" property of beaker host can be used for this
+    on(agent, "export PATH=\"#{agent['privatebindir']}:$PATH\" && #{pxp_agent_dir(agent)}/pxp-agent --version") do |result|
```

Comment:
The modified PATH is only needed for this single on() call. This is the only place in acceptance where pxp-agent is run as an exe rather than a service.
I updated the statement as you suggested, to remove having a separate export statement - thanks very much.